### ISSUE TITLE
non-contiguous tab select

### ISSUE THREAD

**Issue by tpyro** [mbnuqw/sidebery#346]
## Description
I would love the ability to use control- or command-click to select non-contiguous tabs. This would make organizing a large set of tabs very quick and easy.

Thanks!

**Comment by SeerLite** [comment]
Isn't this already a thing with Ctrl?

**Comment by tpyro** [comment]
It's certainly a UI convention but it's not working for me. Shift-click is working for contiguous select. Command-click is definitely not working on Mac and Ctrl-click on PC doesn't seem to be either (I tested inside a VM so I am only slightly less confident due to the key bindings).

**Comment by SeerLite** [comment]
Ah I see. On Mac, Command is `metaKey` while Control is `altKey`. I have no idea what `ctrlKey` would be there then haha

Which one would be more consistent with how multi-select is done everywhere else in Mac-world, Command or Control?

**Comment by tpyro** [comment]
Yes, it's a bit confusing! Command is normally what's used for multi-select. Control typically opens a context menu (equivalent to right-click).
